pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -5539,6 +5539,19 @@ See http://developers.google.com/maps/documentation/utilities/polylinealgorithm`
 			Volatility: tree.VolatilityImmutable,
 		}),
 
+	"st_minimumboundingcircle": makeBuiltin(defProps(),
+		geometryOverload1(func(evalContext *tree.EvalContext, g *tree.DGeometry) (tree.Datum, error) {
+			polygon, _, _, err := geomfn.MinimumBoundingCircle(g.Geometry)
```

Comment:
it looks like we're missing the "num_segs_per_qt_circ" arg here -- see https://postgis.net/docs/ST_MinimumBoundingCircle.html.

Reply:
I have noted this as well, however, didn't find in GEOS implementation where you can control this. Do you have suggestions? 

```c++
extern GEOSGeometry GEOS_DLL *GEOSMinimumBoundingCircle_r(GEOSContextHandle_t handle,
                                                const GEOSGeometry* g, double* radius,
                                                GEOSGeometry** center);
```

Reply:
Looks like I've found how to handle this in [PostGis](https://github.com/postgis/postgis/commit/6a097e28868573af2b2b87084ca809966d5b5dfa#diff-ef8622c09f1c3310d8f5496c7fa4390dd103ea08647ed1ff884ca05bfcdfdf36R6202)

Reply:
I was going to suggest using ST_Buffer on the center by the radius.

Reply:
> I was going to suggest using ST_Buffer on the center by the radius.

Added implementation to do just this.